R/markdown.R

```diff
@@ -68,6 +70,7 @@ markdown <- function(text, tag = NULL, sections = FALSE) {
 #' @keywords internal
 
 markdown_pass1 <- function(text) {
+  rm(list = ls(envir = mddata), envir = mddata)
```

Comment:
Don't we want to cache deps across all calls? In which case, `roxy_meta_get()` might be a better home for this cache?

Reply:
It is fairly fast to do it once, so it is safer not to cache it across all calls, in case the dependencies change.